src/pages/Auth/Signup/Mixin.module.scss

```
@@ -23,33 +23,28 @@ $passive: #c4c4c4;
 }
 
 @mixin font-medium {
-  font-size: 14px;
-  line-height: 18px;
+  font-size: 18px;
+  line-height: 22px;
   font-weight: 500;
 }
 
 @mixin button {
-  position: absolute;
-  bottom: 20px;
   display: flex;
   align-items: center;
   justify-content: center;
-  width: 368px;
-  height: 40px;
+  width: 152px;
+  height: 44px;
   background: $active;
   border: none;
   appearance: none;
-  border-radius: 20px;
-  font-weight: 700;
+  border-radius: 999px;
```

끝을 완전히 둥글게 만들고 싶은거라면 아래처럼 쓸 수 있어요
```suggestion
  border-radius: 100%;
```

---

figma 요청사항에 따라서 그대로 작성했더니 저렇게 표현해버렸네요..ㅎ 수정했습니다